docs/architecture.md

```diff
@@ -0,0 +1,54 @@
+
+# Architecture
+
+## KiliModels
+
+- kiliautoml/models/_base_model.py : Gives an example of a Parent Model class.
```

Comment:
put "[kiliautoml/models/_base_model.py]" to get relative links, ` ` if you want to cite them.